.github/checkgroup.yml

```diff
@@ -163,8 +165,25 @@ subprojects:
       - "lite-cpu (windows-2022, 3.10, latest, stable)"
       - "lite-cpu (windows-2022, 3.7, latest, stable)"
       - "lite-cpu (windows-2022, 3.7, oldest, stable)"
+      - "lightning-lite (GPUs)"
       - "mypy"
-      # TODO: lite should also require (some?) pl checks. this also requires that the path filters are modified
+      # Lite also requires PL checks as it depends on Lite
+      - "pl-cpu (macOS-11, 3.10, latest, stable)"
+      - "pl-cpu (macOS-11, 3.7, latest, stable)"
+      - "pl-cpu (macOS-11, 3.7, oldest, stable)"
+      - "pl-cpu (ubuntu-20.04, 3.10, latest, stable)"
+      - "pl-cpu (ubuntu-20.04, 3.7, latest, stable)"
+      - "pl-cpu (ubuntu-20.04, 3.7, oldest, stable)"
+      - "pl-cpu (windows-2022, 3.10, latest, stable)"
+      - "pl-cpu (windows-2022, 3.7, latest, stable)"
+      - "pl-cpu (windows-2022, 3.7, oldest, stable)"
+      - "make-doctest (pytorch)"
+      - "make-html (pytorch)"
+      - "pytorch-lightning (GPUs)"
```

Comment:
Lite does not support HPU and IPU yet, so I haven't added their checks as required for Lite changes

Please confirm if that's okay @awaelchli. Maybe you want them to still run in each Lite PR for sanity

Reply:
Hmm, we have utilities in Lite that are used by HPU and or IPU on the PL side. If this code would change in a bad way, we wouldn't see these checks fail on the PR, right? The risk is small, so I don't mind. On the other hand, IPU jobs are quite fast, so adding them wouldn't hurt I guess (except when the queue gets long)?

Reply:
The risk is small. If it was up to me, I would not add them until they break and we regret not having added them :stuck_out_tongue: 

I'll leave the decision up to you. Not adding them will mean faster development as you'd have less CI overhead.